examples/latent_sde.py

```diff
@@ -71,24 +71,47 @@ def f(self, t, y):  # Approximate posterior drift.
         inp = torch.cat((torch.sin(t), torch.cos(t), y), dim=-1)
         return self.net(inp)
 
+    def f_aug(self, t, y):
+        y = y[:, 0:1]
+        f, g, h = self.f(t, y), self.g(t, y), self.h(t, y)
+        z = f - h
+        g = torch.where(
+            g.abs() > 1e-7,
+            g,
+            torch.ones_like(g).fill_(1e-7) * g.sign()
+        )
+        z = z / g
+        z = .5 * (torch.norm(z, dim=1, keepdim=True) ** 2.)
+        return torch.cat([f, z], dim=1)
+
     def g(self, t, y):  # Shared diffusion.
         return self.sigma.repeat(y.size(0), 1)
 
+    def g_aug(self, t, y):
+        y = y[:, 0:1]
+        g = self.g(t, y)
+        z = torch.zeros_like(y)
+        return torch.cat([g, z], dim=1)
+
     def forward(self, ts, batch_size, eps=None):
         eps = torch.randn(batch_size, 1).to(self.qy0_std) if eps is None else eps
         y0 = self.qy0_mean + eps * self.qy0_std
+        y0 = torch.cat([y0, torch.zeros(batch_size, 1).to(self.qy0_std)], dim=1)
```

Comment:
Maybe just use `.to(y0)` to save some compute, since the property `qy0_std` is recomputed each time.

Reply:
Done!